pkg/console/install_panels.go

```diff
@@ -653,8 +663,23 @@ func addNetworkPanel(c *Console) error {
 		}
 		logrus.Infof("Network configuration is applied: %s", output)
 
-		c.config.Networks = []config.Network{
-			mgmtNetwork,
+		c.config.Networks = map[string]config.Network{
+			config.MgmtInterfaceName: mgmtNetwork,
+		}
+
+		if mgmtNetwork.Method == config.NetworkMethodDHCP {
+			if addr, err := getIPThroughDHCP(config.MgmtInterfaceName); err != nil {
```

Comment:
If we have reloaded the wicked network configurations, it may not required to get the IP through DHCP here.

Reply:
We better check again because if configuration is not changed, reload doesn't do anything.